python/rikai/types/vision.py

```diff
@@ -190,24 +190,12 @@ def __repr__(self) -> str:
             return "Image(<embedded>)"
         return f"Image(uri={self.uri})"
 
-    def _repr_html_(self):
```

Comment:
Have you tested the full combination of (externalized, embedded) x (Image, ImageDraw ) x (databricks, Jupyter)?

Reply:
I've already tested it for Jupyter. Let me verify the other 4 cases for Databricks.

Reply:
confirmed on Databricks now!